newIDE/app/src/ObjectsList/index.js

```diff
@@ -228,7 +231,7 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
         if (onEditObject) {
           onEditObject(object);
           onObjectCreated(object);
-          onObjectSelected(name);
+          onObjectSelected(name, { object, global: false }); // objectWithContext
```

Comment:
This line might need a comment to say that adding an object is always (at the moment) going to the scene (layout), and not to the project (as a global object) so the context is not global.

Reply:
Hi. I changed this. I added your text as a comment because you explained it good